ui/src/views/network/PublicIpResource.vue

```diff
@@ -136,17 +146,22 @@ export default {
         // VPC IPs don't have firewall
         let tabs = this.$route.meta.tabs.filter(tab => tab.name !== 'firewall')
 
+        if (this.resource.networkofferingconservemode) {
+          this.tabs = tabs
```

Comment:
Seems like resource doesn't have networkofferingconservemode field:
```
![image](https://github.com/apache/cloudstack/assets/10495417/f17403c0-3abf-4c83-8211-268a56ebc14e)

```

Reply:
thanks @Pearl1594 for the review
it should be fixed now